networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/JvmLibP2PNetwork.java

```diff
@@ -158,4 +159,14 @@ public void stop() {
     host.stop();
     scheduler.shutdownNow();
   }
+
+  // Rest API methods
+
+  public String getPeerIDString() {
+    return getPeerId().toBase58();
+  }
+
+  public List<String> getPeersStrings() {
```

Comment:
A similar trick could be employed for this method to simplify it to: `List<T> getPeerIds`

Reply:
getPeerIds only get used in restAPI, so is there a need to do that?

Reply:
not really no